### PR TITLE
Line edit ocamldoc differences

### PR DESCRIPTION
**Pull request by christinerose**
Minor line editing, mostly.

### CODE REVIEW

File: doc/driver.mld [hunk 1]
```diff
@@ -24,28 +24,28 @@ let (>>|=) m f = m >>= fun x -> Ok (f x);;
...
+[odoc] produces output files (HTML or others) in a structured directory tree, so before running [odoc], the output's structure must be decided. For these docs, we want the following structure:
+{ul {- [odoc/index.html] : Main page
+}{- [odoc/{odoc_for_authors.html,...}] : Other documentation pages
 }{- [odoc/odoc_model/index.html] : [odoc] model library subpage
 }{- [odoc/odoc_model/Odoc_model/index.html] : Module page for the module [Odoc_model]
 }{- [odoc/odoc_model/Odoc_model/...] : Further pages for the submodules of [Odoc_model]
-}{- [odoc/odoc_.../index.html] : other [odoc] library pages
-}{- [odoc/deps/stdlib/index.html] : stdlib main page
+}{- [odoc/odoc_.../index.html] : Other [odoc] library pages
+}{- [odoc/deps/stdlib/index.html] : `Stdlib` main page
```
Comment (panglesd) [inline doc/driver.mld:41]:
```suggestion
}{- [odoc/deps/stdlib/index.html] : [Stdlib] main page
```

File: doc/driver.mld [hunk 2]
```diff
@@ -24,28 +24,28 @@ let (>>|=) m f = m >>= fun x -> Ok (f x);;
...
 }{- [odoc/deps/stdlib/Stdlib/index.html] : Module page for the module [Stdlib]
-}{- [odoc/deps/astring/index.html] : astring main page
-}{- [odoc/deps/...] : other dependencies
-}{- [odoc/source/...] : rendered source files
+}{- [odoc/deps/astring/index.html] : `astring` main page
```
Comment (panglesd) [inline doc/driver.mld:43]:
```suggestion
}{- [odoc/deps/astring/index.html] : [astring] main page
```

File: doc/driver.mld [hunk 3]
```diff
@@ -120,7 +120,7 @@ Please note: it's only necessary to link the non-hidden modules (i.e., without a
 {ol {- Generation: odoc html-generate
 }}
 
-Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In this example we output HTML.
+Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In this example we output HTML:
```
Comment (panglesd) [inline doc/driver.mld:123]: I find the `:` confusing. What about

```suggestion
Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In the driver exemplified in this document, we output HTML.
```

File: doc/driver.mld [hunk 4]
```diff
@@ -132,20 +132,20 @@ In this section [odoc] is used to generate the documentation of [odoc] and some
...
+output file from the input because all the files go into the same
 directory.
 
-Odoc supports compiling {i implementations}. This can be useful to render source
+[odoc] supports compiling {i implementations}. This can be useful to render source
 code, generate links from and to source code, and count occurrences of
-identifiers. The [compile-src] command takes a source-tree parent file, and a
-source path which has to be in the source tree.
+identifiers. The [compile-src] command takes a source-tree parent file and a
+source path, which has to be in the source tree.
```
Comment (panglesd) [inline doc/driver.mld:143]: I preferred the previous version: it made more clear that the "has to be in the source tree" refers to the "source path".

**Review by panglesd — APPROVED** [review]
Minor comments inline!
You based your PR on top of #1175, which I only understood after my review.

One thing to know is that we are likely going to deleted the driver.mld file quite soon, so no need to spend too much time on it! When we rewrite it, we'll not forget to ask you for a review! :slightly_smiling_face:

File: doc/driver.mld (hunk 4 above)
Comment (christinerose) [reply doc/driver.mld:143]: So would it be better to say:
"The [compile-src] command takes a source-tree parent file and a
source path that has to be in the source tree."

**Comment by christinerose** [comment]
> Minor comments inline! You based your PR on top of #1175, which I only understood after my review.


Oh no! My mistake! 😦

File: doc/driver.mld (hunk 4 above)
Comment (panglesd) [reply doc/driver.mld:143]: Yes!

File: doc/driver.mld (hunk 3 above)
Comment (christinerose) [reply doc/driver.mld:123]: I guess my comment never sent yesterday!

That was my mistake with the`:` -- I guess I saw a phantom example after it! 🙂 

What about: "For the driver exemplified herein, we output HTML."

**Pull request merged**